app/assets/stylesheets/styles.scss

```diff
@@ -2,4 +2,65 @@
 
 .modal-dialog {
   max-width: 60%;
-}
\ No newline at end of file
+}
+
+.content {
+  max-width: 60%;
+  flex-grow: 1;
+}
+
+.navbar {
+  flex-direction: row;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
General comment on the css:
Please do some nesting. 
`.scale-icon-08` for example doesn't need to exist in a global context. Just put it into the `navbar` class so it exists only in that context. 
Goes for the rest of the CSS classes as well. 
We probably don't want a global `nav-link` class that is not relevant in any other cases.